ansible/playbooks/roles/repository/files/download-requirements/src/config/os_type.py

```diff
@@ -5,7 +5,7 @@
 class OSArch(Enum):
     """ Supported architecture types """
     X86_64 = 'x86_64'
-    ARM64 = 'arm64'
+    ARM64 = 'aarch64'
```

Comment:
Didn't we want to have `arm64` in all of our code instead of `aarch64`? Correct me if I'm wrong.

Reply:
I user `aarch64` places we user ansible_architecture variable which in almalinux systems has value: `aarch64`
![image](https://user-images.githubusercontent.com/48471627/170487191-68c923b0-82b7-40ba-9155-b4146de809cd.png)

![image](https://user-images.githubusercontent.com/48471627/170488466-869637c7-ec5e-4d56-96a6-4d697df5604a.png)

That's why aarch64 is used here.